src/test/java/bio/terra/controller/DatasetOperationTest.java

```diff
@@ -104,7 +106,10 @@
     public void setup() throws Exception {
         createdDatasetIds = new ArrayList<>();
         createdStudyIds = new ArrayList<>();
-        ConnectedOperations.stubOutSamCalls(samService);
+        doNothing().when(samService).createStudyResource(any(), any());
```

Comment:
I think you want to make these changes once to the stubOutSamCalls method in the ConnectedOperations.java. Andrea just refactored into that a few days ago. Her idea was to put that setup in one place rather than cut'n'paste to the tests that needed to mock it.